src/Employer/Employer.Web/Views/Error/PageNotFound.cshtml

```diff
@@ -7,14 +7,12 @@
         <header class="page-header group">
             <h1 class="govuk-heading-xl govuk-!-margin-bottom-6">Page not found</h1>
         </header>
-
         <div class="article-container">
             <article role="article" class="group">
                 <div class="inner">
-                    <p>Please check that you entered the correct web address. Alternatively, <a href="/">return to the homepage</a></p>
+                    <p>Please check that you entered the correct web address. Alternatively, <a asp-route="@RouteNames.Dashboard_Account_Home">return to homepage</a>.</p>
```

Comment:
Same as above.. Return?

Reply:
no because it is in the middle of a sentence.